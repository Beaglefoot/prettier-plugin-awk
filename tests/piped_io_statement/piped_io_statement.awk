{
    print "hi"|"cat"
    print "hi"|&"cat"
    printf("hi")|"cat"
    printf("hi")|&"cat"
}
