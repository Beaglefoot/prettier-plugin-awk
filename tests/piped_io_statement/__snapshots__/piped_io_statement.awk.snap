{
    print "hi"|"cat"
    print "hi"|&"cat"
    printf("hi")|"cat"
    printf("hi")|&"cat"
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    print "hi" | "cat"
    print "hi" |& "cat"
    printf("hi") | "cat"
    printf("hi") |& "cat"
}
